Sources/Logto/Extensions/Collection.swift

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
🤣  multi-line comment?

Reply:
this is generated by Xcode, looks like a convention of swift projects